templates/restart/vpn/node.def.in

```diff
@@ -1,5 +1,5 @@
 help: Restart IPsec VPN
-run: if [ -n "$(cli-shell-api returnActiveValues vpn ipsec)" ]; then
+run: if cli-shell-api existsEffective vpn ipsec; then
```

Comment:
This only restarts ipsec if charon is running. What happens if charon died (unexpectedly), then it won‘t work. 

Can you remove the prep charon path as well?

Reply:
There are 2 daemond `starter` and `charon`  if charon for some reason will be died the started should restart it
```
vyos@r1:~$ ps ax | grep charon
 7900 ?        Ss     0:00 /usr/lib/ipsec/starter --daemon charon
 7901 ?        Ssl    0:00 /usr/lib/ipsec/charon --use-syslog
 8085 pts/0    S+     0:00 grep charon
vyos@r1:~$ 
vyos@r1:~$ sudo kill -9 7901
vyos@r1:~$ 
vyos@r1:~$ ps ax | grep charon
 7900 ?        Ss     0:00 /usr/lib/ipsec/starter --daemon charon
 8089 pts/0    S+     0:00 grep charon
vyos@r1:~$ 
vyos@r1:~$ ps ax | grep charon
 7900 ?        Ss     0:00 /usr/lib/ipsec/starter --daemon charon
 8091 pts/0    S+     0:00 grep charon
vyos@r1:~$ 
vyos@r1:~$ ps ax | grep charon
 7900 ?        Ss     0:00 /usr/lib/ipsec/starter --daemon charon
 8092 ?        Ssl    0:00 /usr/lib/ipsec/charon --use-syslog
 8110 pts/0    R+     0:00 grep charon
vyos@r1:~$ 
vyos@r1:~$ 

```